modules/gdscript/language_server/gdscript_language_server.cpp

```diff
@@ -61,6 +61,9 @@ void GDScriptLanguageServer::_notification(int p_what) {
 		} break;
 
 		case EditorSettings::NOTIFICATION_EDITOR_SETTINGS_CHANGED: {
+			if (GDScriptLanguageServer::port_override != -1) {
```

Comment:
This early return could be merged into the if-block below that checks for remote_port != port. Otherwise changing any of these settings would have no effect until a restart.

Reply:
I instead set the remote_port to port_override if the port_override setting is set, and removed the early return altogether. Let me know if that doesn't work for you though!